subsys/net/lib/mqtt_helper/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library()
+zephyr_include_directories(include)
```

Comment:
Should this be `zephyr_library_include_directories` instead?

Reply:
Seems right (y)

Reply:
Got a build error when using the _library_ flavors.